_docs/tasks/istio-addons.md

```diff
@@ -0,0 +1,257 @@
+---
```

Comment:
not sure if github would do anything better but could you cp + commit + edit in an attempt to see a diff instead of full green ?

Reply:
A diff of what?  The removed stuff ?

This is a fully new doc, so it would just be a red block followed by this long green block.

Reply:
oh there is nothing common between these instructions and the ones that were in quick-start.md ?